actioncable/app/assets/javascripts/action_cable/subscription.coffee

```diff
@@ -44,11 +44,9 @@
 # The "AppearanceChannel" name is automatically mapped between the client-side subscription creation and the server-side Ruby class name.
 # The AppearanceChannel#appear/away public methods are exposed automatically to client-side invocation through the @perform method.
 class ActionCable.Subscription
-  constructor: (@subscriptions, params = {}, mixin) ->
+  constructor: (@consumer, params = {}, mixin) ->
     @identifier = JSON.stringify(params)
     extend(this, mixin)
-    @subscriptions.add(this)
```

Comment:
Where is this logic performed elsewhere, I think we still want to do this? Could be missing something -- not super familiar with CoffeeScript :)